### PR TITLE
Make CI safe to remove external network interface

### PR DESCRIPTION
**Pull request by pwplusnick**
This will add the necessary legacy code for the CI to continue to
operate after the removal of external network interface code.

Note that we are having to test this by running it against the CI, so I expect it to fail at least once.

### CODE REVIEW

File: playbooks/tests/tasks/ci-network.yml [hunk 1]
```diff
@@ -1,8 +1,8 @@
 ---
-- name: determine Neutron external interface name
+- name: deteremine Neutron external interface name
```
Comment (msambol) [inline playbooks/tests/tasks/ci-network.yml]: spelling

**Comment by paulczar** [comment]
wouldn't it make more sense for this to be a feature flagged option?    Opensource users may still want the functionality.

**Comment by pwplusnick** [comment]
@dlundquist ^

**Comment by sandfish8** [comment]
does this mean we are no longer supporting Neutron acting as the gateway for the FIP range and that the cluster will always need to have an External vlan ?

**Comment by pwplusnick** [comment]
@j2sol @dlundquist ^

**Comment by j2sol** [comment]
@sandfish8 in new installs of 2.0 code, the locations those are going to go in are already in that case, where neutron is not acting as the gateway.

We're suggesting feature flagging this for other locations. Also because making code changes like this might be a little /too/ opinionated. So feature flag seems appropriate.

File: playbooks/tests/tasks/ci-network.yml [hunk 2]
```diff
@@ -10,7 +10,7 @@
   failed_when: False
 
 - name: set neutron interface fact
-  set_fact: neutron_external_interface=brq{{ result.id|truncate(length=11,killwords=true,end='') }}
+  set_fact: neutron_external_interface=brq{{ result.id|truncate(lenght=11,killwords=true,end='') }}
```
Comment (paulczar) [inline playbooks/tests/tasks/ci-network.yml]: WAT

Comment (pwplusnick) [inline playbooks/tests/tasks/ci-network.yml]: I just can't spell... :(

**Comment by pwplusnick** [comment]
This PR is replaced by PR #1402

**Pull request closed without merging**